lua/types/config.lua

```diff
@@ -872,53 +1051,91 @@
----whether numbering starts with `0` or `1`
+---whether numbering starts with `0` or `1`.
+---
 ---@field show_tab_index_in_tab_bar? boolean
 ---When set to `true` (the default), the tab bar will display the tabs
 ---associated with the current window.
 ---
----When set to `false`, the tabs will not be drawn into the tab bar
+---When set to `false`, the tabs will not be drawn into the tab bar.
+---
 ---@field show_tabs_in_tab_bar? boolean
 ---@field ssh_backend? SshBackend
 ---@field ssh_domains? SshDomain[]
 ---If true, the `Backspace` and `Delete` keys generate `Delete` and `Backspace`
 ---keypresses, respectively, rather than their normal keycodes.
 ---On macOS the default for this is true because its Backspace key
 ---is labeled as Delete and things are backwards.
+---
 ---@field swap_backspace_and_delete? boolean
 ---@field switch_to_last_active_tab_when_closing_tab? boolean
 ---If `true`, show_tab_index_in_tab_bar uses a zero-based index.
 ---
----The default is `false` and the tab shows a one-based index
+---The default is `false` and the tab shows a one-based index.
+---
 ---@field tab_and_split_indices_are_zero_based? boolean
 ---When `config.tab_bar_at_bottom = true`, the tab bar will be rendered
 ---at the bottom of the window rather than the top of the window.
 ---
----The default is `false`
+---The default is `false`.
+---
 ---@field tab_bar_at_bottom? boolean
 ---@field tab_bar_style? TabBarStyle
 ---Specifies the maximum width that a tab can have in the tab bar
 ---when using retro tab mode.
 ---It is ignored when using fancy tab mode.
 ---
----Defaults to `16` glyphs in width
+---Defaults to `16` glyphs in width.
+---
 ---@field tab_max_width? number
----What to set the `$TERM` variable to
+---What to set the `$TERM` variable to.
+---
 ---@field term? string
+---@field text_blink_rapid_ease_in? EasingFunction
+---@field text_blink_rapid_ease_out? EasingFunction
+---Specifies how often blinking text (normal speed) transitions
+---between visible and invisible, expressed in milliseconds.
+---
+---Setting this to `0` disables slow text blinking.
+---
+---Note that this value is approximate due to the way
+---that the system event loop schedulers manage timers;
+---non-zero values will be at least the interval specified
+---with some degree of slop.
+---
+---@field text_blink_rate? integer
+---Specifies how often blinking text (rapid speed) transitions
+---between visible and invisible, expressed in milliseconds.
+---
+---Setting this to `0` disables rapid text blinking.
+---
+---Note that this value is approximate due to the way
+---that the system event loop schedulers manage timers;
+---non-zero values will be at least the interval specified
+---with some degree of slop.
+--- ---
+---Note that blinking is no longer a binary blink, but interpolates
+---between invisible and visible text using an easing function.
+---See `text_blink_rapid_ease_in` and `text_blink_rapid_ease_out`
+---for more information.
```

Comment:
I don't know what to do about these 'Notes' parts where Wez is giving an update about how things have changed from previous versions. I've just been incorporating those notes into the docs because I don't like having to read all the way to bottom of the documentation page to see what the 'latest' change to this option/value is, I'd rather see that up front. My opinion is that when we see these sort of 'update for version 2024...' additions, we just incorporate those notes and adjust the annotations/docs accordingly. What do you think? This can be a TODO for sometime later, I don't want to hold up this PR for it!

Reply:
Yeah, the trick's to integrate these _"version updates"_ seamlessly in the annotations.

I found for now just to add a **horizontal ruler** (`---`) just as a compromise,
but if we can manage to find a better solution I'm all for it.